swap/src/config.rs

```diff
@@ -104,7 +122,7 @@ mod regtest {
 
     pub static MONERO_FINALITY_CONFIRMATIONS: u32 = 1;
 
-    pub static BITCOIN_CANCEL_TIMELOCK: Timelock = Timelock::new(50);
+    pub static BITCOIN_CANCEL_TIMELOCK: Timelock = Timelock::new(100);
```

Comment:
Do we know what this has to be higher now (compared to master, where the tests pass)?

Reply:
It's not relevant anymore as we use custom times for each test.